src/standalone/Soil/energy_hydrology.jl

```diff
@@ -987,8 +987,6 @@ function soil_turbulent_fluxes_at_a_point(
         )
         x::FT = 4 * K_sfc * (1 + Ẽ0 / (4 * K_c))
         Ẽ *= x / (Ẽ0 + x)
-    else
-        Ẽ *= 0 # condensation, set to zero
```

Comment:
Is it okay to leave evap/sublimation unchanged if condensation is occurring? Maybe in this case they'll tend towards 0 in the equations solved in `evaporation` anyway?

Reply:
let's discuss in person!